django_project/ford3/views/provider.py

```diff
@@ -63,6 +69,11 @@ def edit_provider(request, provider_id):
             'provider_id': provider_id,
             'is_new_provider': provider.is_new_provider,
         }
+
+        # use case: load page for the first time
+        if provider.provider_logo:
+            context['provider_logo'] = '/ford3' + provider.provider_logo.url
```

Comment:
Use url reverse, or url join.

Reply:
What is the `/ford3` url? It's best to not hardcode it and take it from settings or some customizable variables.